test/util/fake-xml-http-request-test.js

```diff
@@ -2304,9 +2317,11 @@ if (typeof window !== "undefined") {
 
             it("event listeners can be removed", function () {
                 var callback = function () {};
+
+                var eventListenersCount = this.xhr.upload.eventListeners.load.length;
                 this.xhr.upload.addEventListener("load", callback);
                 this.xhr.upload.removeEventListener("load", callback);
-                assert.equals(this.xhr.upload.eventListeners.load.length, 0);
+                assert.equals(this.xhr.upload.eventListeners.load.length, eventListenersCount);
```

Comment:
Could you add a comment explaining this change? My immediate reaction was "This is making the test less explicit". 

Also since we can no longer count on zero listeners, can you change the callback and trigger the event to confirm it is not called?